packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -105,6 +105,23 @@ const DUPLICATED_MAC_ADDRESSES_FILTERS = {
   filterOnlyRunningVms: 'nRunningVms:>1',
 }
 
+const DEFAULT_LOCAL_SRS_COLUMNS = [
+  {
+    name: _('pool'),
+    itemRenderer: pool => pool.name_label,
+    sortCriteria: pool => pool.name_label,
+  },
+  {
+    name: _('sr'),
+    itemRenderer: pool => <Sr container={false} id={pool.default_SR} link spaceLeft={false} />,
+  },
+  {
+    name: _('host'),
+    itemRenderer: (pool, { hosts, srs }) => <Host id={hosts[srs[pool.default_SR].$container].id} pool={false} link />,
```

Comment:
You don't need to get the host object since all you need is the ID.

Reply:
I don't think we should hide the pool. Maybe even make it its own column, but at least leave it in the host's render.

Reply:
Pool already have his own column. [code](https://github.com/vatesfr/xen-orchestra/pull/6033/files#diff-2af4feb739063d24cfb951db648985938b2292a9e5edc862aec3065b6d8a74faR110-R112)